docker-compose.yml

```diff
@@ -1,13 +1,18 @@
 version: "3"
 
 services:
-  list-app:
-    build: ./list-app
-
   api:
     build: ./api
+    container_name: api
+    ports:
+      - 8080:3000
+    depends_on:
+      - db
 
   db:
     image: mongo
     container_name: db
+    ports:
+      - 27017:27017
     volumes:
+      - ./api/data/db:/data/db
```

Comment:
I'm not sure where is the best location for the app data persistence... I was doubting between this one and `/data/db`. What do you think?